galaxy_api/api/ui/serializers/namespace.py

```diff
@@ -36,6 +38,21 @@ class Meta:
             'resources'
         )
 
+    def validate_name(self, name):
+        if not name:
+            raise ValidationError(detail={
+                'name': "Attribute 'name' is required"})
+        elif not re.match(r'^[a-zA-Z0-9_]+$', name):
```

Comment:
Minor: Because each of these clauses raises an exception, `elif` is not really needed.